src/app.js

```diff
@@ -18,7 +19,27 @@ library.add(faSignInAlt, faSignOutAlt, faCloudDownloadAlt, faCloudUploadAlt);
 
 Vue.component('font-awesome-icon', FontAwesomeIcon);
 
-Vue.use(Vuetify);
+Vue.use(Vuetify, {
+  theme: {
+    primary: "#F26521",
+    defaultPrimary: "#1976D2",
+    discordOrange: "#F26521",
+    discordPurple: "#7289DA",
+    crlogo: colors.blueGrey.darken2,
+    crlogoDark: colors.blueGrey.darken2,
+    crlogoLight: colors.blueGrey.lighten5,
+    background: colors.grey.darken4,
+    background2: colors.grey.darken4,
+    search: colors.grey.darken4,
+    backgroundDark: colors.grey.darken4,
+    backgroundLight: colors.grey.lighten2,
+    background2Light: colors.grey.lighten4,
```

Comment:
Are there not already colors like this as part of the Vuetify default theme? I feel like we should just be able to use what's already there, and override that if it's not sufficient.